lib/event_sourcery/postgres/reactor.rb

```diff
@@ -74,7 +74,11 @@ def emit_event(event_or_hash, &block)
 
       def invoke_action_and_emit_event(event, action)
         action.call(event.body) if action
```

Comment:
I wondered whether to validate before `action.call` but the action often mutates the state of the event so it may pass validation afterwards when it wouldn't have before.